galcem/plottingtool.py

```diff
@@ -78,7 +78,7 @@ def tracked_elements_3D_plot(self, cmap_name='magma_r', cbins=10): # angle = 2 *
         smap = ax.scatter(x,y, marker='s', alpha=0.95, edgecolors='none', s=5, cmap=cmap_name, c=percent_colors) 
         smap.set_clim(0, 100)
         norm = colors.Normalize(vmin=0, vmax=100)
-        cb = fig.colorbar(cm.ScalarMappable(norm=norm, cmap=cmap_name), orientation='vertical', pad=0.0)
+        cb = fig.colorbar(cm.ScalarMappable(norm=norm, cmap=cmap_name), orientation='vertical', pad=0.0, ax=ax)
```

Comment:
This may only be required since I used a newer version of matplotlib. Happy to supply the conda env which I used to get the MWE working